addons/sms/static/src/messaging/component/message/message.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-inherit="mail.messaging.component.Message" t-inherit-mode="extension">
+        <xpath expr="//*[@name='failureIcon']" position="replace">
+            <t t-if="message.message_type === 'sms'">
+                <i class="o_Message_notificationIcon fa fa-mobile"/> SMS
+            </t>
+            <t t-else="">$0</t>
+        </xpath>
+
+        <xpath expr="//*[@name='notificationIcon']" position="replace">
+            <t t-if="message.message_type === 'sms'">
+                <i class="o_Message_notificationIcon fa fa-mobile"/> SMS
+            </t>
+            <t t-else="">$0</t>
+        </xpath>
```

Comment:
Would we want to let the message entity return from a compute/getter "fa fa-mobile" depending on its type so that no template override is needed here?

Reply:
no